server/server.js

```diff
@@ -183,6 +186,33 @@ app.get(
   })
 );
 
+app.get(
+  `${apiBaseUrl}/participants/confirm-interest`,
+  asyncMiddleware(async (req, res) => {
+    const confirmationId = req.query.id;
+    const isValid = uuidValidate(confirmationId) && (await validateConfirmationId(confirmationId));
+
+    if (isValid) {
+      return res.status(isValid ? 200 : 400).send();
```

Comment:
this ternary operator is redundant.
```suggestion
      return res.status(200).send();
```